data/human/human missions.txt

```diff
@@ -7455,3 +7455,1836 @@ mission "Adelita 5"
+				goto greenoptions
+
+			label getgear4
+			action
+				set "suit2"
+				"timothy radrickson suit" = 1
+				payment -250000
+			`	The proprietor quickly rings you up as if worried you'll change your mind at any moment, and gleefully takes your <payment>.`
+			`	"Safe travels, privateer," he says as you leave his stall.`
+				goto greenoptions
+
+			label med1
+			action
+				set "med1"
+			`	There are many stalls loudly advertising all kinds of illegal drugs, and it takes some work to sift through them until you come across one selling legal drugs as well. The man running it is sallow and pock-marked, but his eyes are keen. He almost hisses as he greets you.`
+			`	"Whatever you need, wherever you're going, I have you covered."`
+			choice
+				`	(Ask about detoxification medicine for Oblivion.)`
+					goto askdrugs
+				`	(Leave.)`
+
+			`	This is not the place for you. You take your leave and return to the emporium at large.`
+				goto greenoptions
+
+			label askdrugs
+			`	The drug dealer flashes you with his yellow teeth and unnerving smile. "For every kind of toxic environment under any sun." He jolts towards his little wall of drawers, pulling out one labeled "Oblivion detoxifiers." From the drawer he picks a bottle up, checking its label.`
+			`	"It says it expired a few years ago, but that's just so the corpos can squeeze the common man, ya know? Stuff's still good." He shrugs.`
+			`	"So what'dya say? It'll cost'ya a cool 50,000 credits. But trust me, that's way cheaper than what you'll find elsewhere."`
+			choice
+				`	(Buy the expired detoxification medicine for Tim.)`
+					goto getmed
+				`	(Pass.)`
+
+			`	The drug-dealer says as you go away, "You won't find a better deal than this. I'll be here if you change your mind."`
+				goto greenoptions
+
+			label med2
+			action
+				set "med2"
+			`	The drug-dealer's knowing smile is almost a sneer as you return to his stall. He waits to see if you're ready to buy something.`
+			choice
+				`	(Buy the expired detoxification medicine for Tim.)`
+					goto getmed
+				`	(Pass.)`
+
+			`	You leave the drug-dealer behind and head back into the emporium.`
+				goto greenoptions
+			
+			label getmed
+			action
+				set "med2"
+				"timothy radrickson	medicine" = 2
+				payment -50000
+			`	"Trust me, it's still good," the drug-dealer says as he takes your <payment>. He hands you the drugs with a smirk.`
+				goto greenoptions
+
+			label end
+			action
+				clear "med1"
+				clear "med2"
+				clear "suit1"
+				clear "suit2"
+				clear "id1"
+				clear "id2"
+			`	It's never wise to linger in a place like this, especially aimlessly. With your business concluded you leave Greenrock's stalls behind.`
+				decline
+
+
+mission "Timothy Radrickson 5e: Back to Oblivion"
+	source Oblivion
+	to offer
+		has "Timothy Radrickson 5a: Timshank Redemption: done"
+	on offer
+		"timothy radrickson preparedness" = "timothy radrickson	medicine" + "timothy radrickson suit" + "timothy radrickson id"
```

Comment:
Oddly enough this doesn't work in-game if it's not alphabetical

![image](https://github.com/user-attachments/assets/f3ea0742-8225-451d-a8b7-e7c753753fda)

```suggestion
		"timothy radrickson zpreparedness" = "timothy radrickson	medicine" + "timothy radrickson suit" + "timothy radrickson id"
```

Reply:
I'll take your word for it; I haven't heard of this issue before.